### ISSUE TITLE
Returning result: Exception training model: We couldn't connect to 'https://huggingface.co'

### ISSUE THREAD

**Issue by ZeroCool22** [d8ahazard/sd_dreambooth_extension#524]
![Screenshot_11](https://user-images.githubusercontent.com/13344308/207493439-2002d6e1-2af8-4933-a84e-2a6f18ca9fa2.jpg)

`
Python revision: 3.10.6 (tags/v3.10.6:9c7b4bd, Aug  1 2022, 21:53:49) [MSC v.1932 64 bit (AMD64)]
Dreambooth revision: 9e3584f0edd2e64d284b6aaf9580ade5dcceed9d
SD-WebUI revision: 685f9631b56ff8bd43bce24ff5ce0f9a0e9af490

Checking Dreambooth requirements...
[+] bitsandbytes version 0.35.0 installed.
[+] diffusers version 0.10.2 installed.
[+] transformers version 4.25.1 installed.
[+] xformers version 0.0.14.dev0 installed.
[+] torch version 1.12.1+cu113 installed.
[+] torchvision version 0.13.1+cu113 installed.
`

**db_config.json**

{
    "adam_beta1": 0.9,
    "adam_beta2": 0.999,
    "adam_epsilon": 1e-08,
    "adam_weight_decay": 0.01,
    "attention": "xformers",
    "center_crop": false,
    "concepts_path": "",
    "epoch_pause_frequency": 0.0,
    "epoch_pause_time": 60.0,
    "gradient_accumulation_steps": 1,
    "gradient_checkpointing": true,
    "half_model": false,
    "hflip": true,
    "learning_rate": 2e-06,
    "lora_learning_rate": 0.0002,
    "lora_txt_learning_rate": 0.0002,
    "lr_scheduler": "constant",
    "lr_warmup_steps": 0,
    "max_token_length": 75,
    "max_train_steps": 4500,
    "mixed_precision": "fp16",
    "model_dir": "C:\\Users\\ZeroCool22\\Desktop\\Auto\\models\\dreambooth\\butaska",
    "model_name": "butaska",
    "not_cache_latents": true,
    "num_train_epochs": 1,
    "pad_tokens": true,
    "pretrained_model_name_or_path": "C:\\Users\\ZeroCool22\\Desktop\\Auto\\models\\dreambooth\\butaska\\working",
    "pretrained_vae_name_or_path": "C:\\Users\\ZeroCool22\\Desktop\\Nueva\\vae-ft-mse-840000-ema-pruned.ckpt",
    "prior_loss_weight": 1,
    "resolution": 512,
    "revision": 0,
    "sample_batch_size": 1,
    "save_class_txt": true,
    "save_embedding_every": 0,
    "save_preview_every": 0,
    "save_use_global_counts": true,
    "save_use_epochs": false,
    "scale_lr": false,
    "src": "C:\\Users\\ZeroCool22\\Desktop\\Auto\\models\\Stable-diffusion\\v1-5-pruned-emaonly.ckpt",
    "shuffle_tags": false,
    "train_batch_size": 1,
    "train_text_encoder": false,
    "use_8bit_adam": true,
    "use_concepts": false,
    "use_cpu": false,
    "use_ema": false,
    "use_lora": false,
    "scheduler": "euler-ancestral",
    "v2": false,
    "has_ema": "False",
    "concepts_list": [
        {
            "max_steps": -1,
            "instance_data_dir": "C:\\Users\\ZeroCool22\\Desktop\\Putaska",
            "class_data_dir": "",
            "instance_prompt": "butaska",
            "class_prompt": "",
            "save_sample_prompt": "",
            "save_sample_template": "",
            "instance_token": "",
            "class_token": "",
            "num_class_images": 0,
            "class_negative_prompt": "deformities, lowres, extra legs, extra arms, bad anatomy, bad hands, text, error, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality, normal quality, jpeg artifacts, signature, watermark, username, blurry",
            "class_guidance_scale": 7.5,
            "class_infer_steps": 40,
            "save_sample_negative_prompt": "deformities, lowres, extra legs, extra arms, bad anatomy, bad hands, text, error, missing fingers, extra digit, fewer digits, cropped, worst quality, low quality, normal quality, jpeg artifacts, signature, watermark, username, blurry",
            "n_save_sample": 1,
            "sample_seed": -1,
            "save_guidance_scale": 7.5,
            "save_infer_steps": 40
        }
    ],
    "lifetime_revision": 0
}

**-All updated to the latest version.**

**-OS: Windows 10 x64.**
**-GPU: 1080 TI.**

**Comment by MrZhongFF** [comment]
the same issue

**Comment by yanfeiwong** [comment]
same here....... and the v2.1 training model fale to crate... Exception while extracting model: shape '[4096, 1024]' is invalid for input of size 1857411

**Comment by d8ahazard** [comment]
Using a custom VAE is currently only supported when the vae is saved in diffusers format - i.e. "pytorch_model.bin".
